etl/helpers.py

```diff
@@ -263,3 +267,405 @@ def load_dependency(
             dataset = catalog.Dataset(dataset_path)
 
         return dataset
+
+
+# For backwards compatibility.
+Names = PathFinder
```

Comment:
We've stopped using `Names`. There shouldn't be any more `Names` in ETL (and if there are, please replace them with `PathFinder`)